src/v/cluster/partition_balancer_planner.cc

```diff
@@ -1275,11 +1275,17 @@ void partition_balancer_planner::reassignable_partition::revert(
       _reallocated->partition.is_original(move.previous()->node_id),
       "ntp {}: move {}->{} should have been from original node",
       _ntp,
-      move.current(),
-      move.previous());
+      move.previous(),
+      move.current());
```

Comment:
not confusing at all!

Reply:
It's an assert so I guess nobody noticed :man_shrugging: no idea why I wrote it like this in the first place